src/spec/models/tag_spec.rb

```diff
@@ -1,5 +1,7 @@
 require 'rails_helper'
 
 RSpec.describe Tag, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  subject { create :tag }
+
+  it { is_expected.to have_and_belong_to_many :functions }
```

Comment:
we'll need more validation for the proper labels names like what we did for functions names